safe/datastore/folder.py

```diff
@@ -0,0 +1,188 @@
+# coding=utf-8
+"""
+InaSAFE Disaster risk assessment tool by AusAid
+
+Contact : ole.moller.nielsen@gmail.com
+
+.. note:: This program is free software; you can redistribute it and/or modify
+     it under the terms of the GNU General Public License as published by
+     the Free Software Foundation; either version 2 of the License, or
+     (at your option) any later version.
+
+"""
+
+from os.path import splitext
+from os import remove
+from PyQt4.QtCore import QFileInfo, QDir
+from qgis.core import (
+    QgsVectorFileWriter,
+    QgsRasterLayer,
+    QgsRasterPipe,
+    QgsRasterFileWriter
+)
+
+from safe.datastore.datastore import DataStore
+from safe.common.exceptions import ErrorDataStore, ShapefileCreationError
+
+EXTENSIONS = ('shp', 'asc', 'tiff', 'tif')
+
+
+class Folder(DataStore):
+    """
+    Folder DataStore
+
```

Comment:
A folder based data store is a collection of shape files and tiff images stored in a common folder.